### DISCUSSION TITLE
how can i fix this Authentication Failure

### DISCUSSION THREAD

**Discussion by pamodaliyanage** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#701] [Q&A]
when i save my key file  path it says
 (The Key file provided is not valid. Make sure the file is of the form "client_secret_<your_id>.apps.googleusercontent.com.json"
Error: Invalid Key file)
![image](https://user-images.githubusercontent.com/98894195/164883368-632bf37c-2c67-4634-ae03-4c7e53bc3577.png)

**Comment by sumeshir26** [comment]
#671
